lxd/identity/cache.go

```diff
@@ -0,0 +1,151 @@
+package identity
+
+import (
+	"crypto/x509"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/canonical/lxd/shared"
+	"github.com/canonical/lxd/shared/api"
+)
+
+// Cache represents an thread-safe in-memory cache of the identities in the database.
+type Cache struct {
+	entries map[string]map[string]CacheEntry
+	mu      sync.RWMutex
+}
+
+// CacheEntry represents an identity. The Certificate field is pre-computed for identities with AuthenticationMethod
```

Comment:
Can you move the "The Certificate field..." bit onto a new line so it doesnt break the sentence.